core/camel-core-languages/src/main/docs/modules/languages/pages/simple-language.adoc

```diff
@@ -18,10 +18,10 @@ being a very small and simple language for evaluating
 `Expression` or `Predicate` without requiring any new dependencies
 or knowledge of other scripting languages such as Groovy.
 
-The simple language is designed with intend to cover almost all the common use cases
+The simple language is designed to intend to cover almost all the common use cases
 when little need for scripting in your Camel routes.
 
-However, for much more complex use cases then a more powerful language is recommended such as:
+However, for much more complex use cases, then a more powerful language is recommended such as:
```

Comment:
Remove `then` -> `for much more complex use cases, a more powerful language is recommended`